crates/typst-docs/src/lib.rs

```diff
@@ -34,7 +34,7 @@ static CATEGORIES: Lazy<yaml::Mapping> = Lazy::new(|| yaml("reference/categories
 static GROUPS: Lazy<Vec<GroupData>> = Lazy::new(|| yaml("reference/groups.yml"));
 
 static LIBRARY: Lazy<Prehashed<Library>> = Lazy::new(|| {
-    let mut lib = typst_library::build();
+    let mut lib = typst_library::build(Default::default());
```

Comment:
Clippy doesn't like `Default::default`. It wants you to include the actual type name.

Reply:
Really ? that's a bit silly. I didn't get that lint, did you do something special to get it ?

Reply:
https://rust-lang.github.io/rust-clippy/stable/index.html#/default_trait_access

Reply:
From that link, seems like it is restricted to when `pedantic` lints are enabled. Typst doesn't enable those on CI, so I'm not sure we should follow those.

With that said, changing to `SysArguments::default()` probably would aid readability anyway.

Reply:
> probably would aid readability anyway

That's the idea of pedantic lints. The code does exactly the same thing, just a bit clearer because people may be confused about "the default of _what_?" when they see this.